### DISCUSSION TITLE
Git tool getting an error that doesn't seem to be an error??

### DISCUSSION THREAD

**Discussion by RandyHoffpauir** [thoughtspot/cs_tools#131] [General] [answered]
When I run this command:
     cs_tools.exe tools git branches commit --tag mytag --branch-name mybranch --comment "my comment" --delete-aware
As part of the debug log I am getting this error:
           DEBUG    <<< [20:58:56] HTTP 403 <-                   _client.py:106
                    /callosum/v1/tspublic/v1/session/orgs                      
                    (0.109424s)                                                
                    {"type":"Forbidden","description":"The                     
                    client does not have access rights to the                  
                    content; that is,it is unauthorized, so the                
                    server is refusing to give the requested                   
                    resource. Unlike 401, the client\u0027s                    
                    identity is known to the server."}           

This error shows just before the actual commit command is run. 
I am getting the same error when I run this command:
     cs_tools.exe tools git branches validate dev main

Are these actual errors? The commands seem to run and complete successfully.

**Comment by boonhapus** [comment]
@RandyHoffpauir do you have orgs enabled in your ThoughtSpot cluster?

If not, can you share the last few log files to **[this secure fileshare link](https://thoughtspot.egnyte.com/ul/nk4FP3bkpA)**? This will allow me to debug why you're seeing the issue.

`cs_tools logs report --latest 5`

**Reply by RandyHoffpauir** [reply]
We do not have orgs enabled in our ThoughtSpot cluster.
I sent you the files.

**Reply by boonhapus** [reply, marked as answer]
Gotcha, thanks @RandyHoffpauir ! It's fine to ignore this `DEBUG` log. This is actually me checking to see if orgs is enabled. ThoughtSpot doesn't reliably expose an attribute on the session/info payload to see if orgs is enabled so my past self must have thought "I'll just call an orgs-only API and see if it fails". 😅 

https://github.com/thoughtspot/cs_tools/blob/13df525ba945a366f91fdb8a9db004b1aadd4521/cs_tools/thoughtspot.py#L165

If it's concerning, I can find a way to filter it out.

**Reply by RandyHoffpauir** [reply]
If that is all it is, then I am fine with it. Just wanted to make sure.
